spark-jobs/src/main/scala/za/co/absa/enceladus/common/PathConfig.scala

```diff
@@ -0,0 +1,3 @@
+package za.co.absa.enceladus.common
```

Comment:
Please add the Copyright

Reply:
It's added in [here](https://github.com/AbsaOSS/enceladus/pull/1377/commits/9fbb264c2ae01c881159a5488cdee0b0713dcb11)